akka-cluster/src/main/resources/reference.conf

```diff
@@ -54,6 +54,11 @@ akka {
     # `akka.cluster.DowningProvider` having a public one argument constructor accepting an `ActorSystem`
     downing-provider-class = ""
 
+    # Artery only setting
+    # When a node has been gracefully downed, let this time pass (to allow for example
+    # cluster singleton handover to complete) and then quarantine the downed node.
+    quarantine-downed-node-after=30s
```

Comment:
`quarantine-removed-node-after`

downing is for the unexpected crash scenario, leaving is for the graceful removal